test/katex-spec.js

```diff
@@ -2976,7 +2976,7 @@ describe("Unicode", function() {
     });
 
     it("should parse relations", function() {
-        expect("∈∋∝∼∽≂≃≅≈≊≍≎≏≐≑≒≓≖≗≜≡≤≥≦≧≫≬≳≷≺≻≼≽≾≿∴∵∣≔≕⩴").toParse();
+        expect("∈∋∝∼∽≂≃≅≈≊≍≎≏≐≑≒≓≖≗≜≡≤≥≦≧≫≬≳≷≺≻≼≽≾≿∴∵∣≔≕⩴≪⋘").toParse();
```

Comment:
Can you put `≪` beside the `≫`?  There should also be `\ggg` can you add the unicode char for that beside the `⋘`?

Reply:
Done.